test/unittest/tests/block/test_block_ckpt.cpp

```diff
@@ -36,7 +36,9 @@ TEST_CASE("Block helper: __wt_rduppo2", "[block]")
 static void
 test_ckpt_add_blkmod_entry(wt_off_t offset, wt_off_t len, uint64_t expectedBits)
 {
-    std::shared_ptr<SessionWrapper> session = SessionWrapper::buildTestSessionWrapper();
+    auto conn = ConnectionWrapper();
```

Comment:
I'd replace this with `ConnectionWrapper conn;` as that's easier to read and more typical C++. In the current code, unless someone knows that ConnectionWrapper is a class, this could look like a function call.

Reply:
Fair point, thanks. Fixed in 6e9f46b